Library/Homebrew/download_strategy.rb

```diff
@@ -1200,6 +1200,43 @@ def update
   end
 end
 
+# GCSDownloadStrategy downloads files from Google Cloud Storage
```

Comment:
This documentation will have to be elsewhere, as well…

I'm not sure where but wait to do so until some project members take a look at this.